src/utils/crypto.js

```diff
@@ -158,6 +158,9 @@ exports.default = {
             result.set([randomValue], i);
         }
         return result;
-    }
+    },
+    hashChain(input) {
+    return keccak(blake2b(input));
+}
```

Comment:
This `hashChain` function is duplicated. The `hashChain` is already defined in Line 34.